charts/pega/values-large.yaml

```diff
@@ -125,6 +125,12 @@ global:
       hpa:
         enabled: true
 
+      # Set enabled to true to include a Pod Disruption Budget for this tier
```

Comment:
Set enabled to true to include a Pod Disruption Budget for this tier. To enable this budget, specify either a `pdb.minAvailable` or `pdb.maxUnavailable` value and comment out the other parameter.

Replicate this wording throughout the values.yaml files. Right now, I think it's a bit too un-documented on what I have to do to make it work for maxUnavailable.